cocos/animation/core/pose-allocator.ts

```diff
@@ -65,6 +65,11 @@ export class PoseAllocator {
         // This does not cause big problem since all pose allocators share the same stack memory.
     }
 
+    get top () {
```

Comment:
How about `back` or `tail` ? likes https://learn.microsoft.com/zh-cn/cpp/standard-library/vector-class?view=msvc-170#back

Reply:
I feel top is more comprehensive: the stack in brain should likely be a top-to-down structure and the new coming element is placed on top of the stack. C++ `std::stack` is also using this term.

Reply:
Seems no where describes this is a stack allocator, so I can't relate the context. Should change the name of the allocator to `PoseStackAllocator`?

Reply:
Renamed as your suggested. https://github.com/cocos/cocos-engine/pull/14779/commits/dcf00b7ed67e628d183a77197dd3fd1686b5c2c9